packages/amplify-cli-core/src/state-manager/pathManager.ts

```diff
@@ -24,6 +24,7 @@ export const PathConstants = {
 
   // resource level
   BuildDirName: 'build',
+  SchemaSnapshot: '.snapshot',
```

Comment:
As this stands in the `PathConstants`, it looks like it might be reusable, given it's only used in `getBackendSnapshotVFSPath` I might just make inline there so nobody else uses it misunderstanding why it's defined..